utils/build-script

```diff
@@ -1186,7 +1186,7 @@ Using option presets:
   using the name=value syntax on the command line.
 
 
-Any arguments not listed are forwarded directly to Swift's
+Any arguments not listed, or arguments after `--` are forwarded to Swift's
```

Comment:
Actually, this is not true.
https://github.com/apple/swift/blob/master/utils/swift_build_support/swift_build_support/migration.py#L33-L34
`--` is simply ignored.